pkgs/by-name/cr/create-dmg/package.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "create-dmg";
+  version = "v1.2.2";
```

Comment:
Version should start with a digit.

Reply:
When changging this, also change the revision on `fetchFromGitHub` to:
`rev = "refs/tags/v${finalAttrs.version}";`